go.json

```diff
@@ -23,8 +23,8 @@
         "maximum": ""
     },
     "postInstall": {
-        "windows": ["go", "get", "github.com/getgauge-contrib/gauge-go"],
-        "linux": ["go", "get", "github.com/getgauge-contrib/gauge-go"],
-        "darwin": ["go", "get", "github.com/getgauge-contrib/gauge-go"]
```

Comment:
Will be the `@latest` suffix rewritten to actual version when a release is done?